e2e_test/extended_mode/basic.slt

```diff
@@ -78,18 +68,3 @@ with t as (select generate_series(1,3,1)) select * from t;
 1
 2
 3
-
-query III
-select 42::smallint, 42::int, 42::bigint;
-----
-42 42 42
-
-query III
-select 42::real,42::double precision,42::decimal;
-----
-42 42 42
-
-query TTT
-select '20:55:12'::time,'2022-07-12'::date,'2022-07-12 20:55:12'::timestamp;
-----
-20:55:12 2022-07-12 2022-07-12 20:55:12
```

Comment:
Hmm why do we remove these?

Reply:
We have cover it with extended_mode/type.slt.🤔